### DISCUSSION TITLE
Support for OpenVINO

### DISCUSSION THREAD

**Discussion by jet10000** [postgresml/postgresml#1014] [General]
🙏

**Comment by montanalow** [comment]
OpenVINO does a lot of things with overlapping functionality in some of the areas already covered by PostgresML. What are the features you're interested in, or what's your particular use case like?

**Reply by jet10000** [reply]
for intel GPU

**Reply by jet10000** [reply]
https://github.com/intel/scikit-learn-intelex

**Reply by montanalow** [reply]
Cool, this is not a big lift, although it seems more applicable to people using a database locally, rather than in a datacenter, since most hosted intel chips do not include the onboard GPU. I'd love to understand your local database use case more, to see if there are other features we could address.
